reseq_ros2/config/reseq_mk1_can.yaml

```diff
@@ -11,6 +11,9 @@ modules:
     hasJoint: true
     hasEndEffector: false
 realsense_config: "realsense_rgb_motion.yaml"
+sensors:
+  - realsense: true
+  - lidar: true
```

Comment:
These should be added to the VCAN files (at least the empty list otherwise)
```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): 'sensors'
```